Windows/windows/ingest/parser.yml

```diff
@@ -1068,3 +1112,13 @@ stages:
       - set:
           source.address: "{{json.event.SourceHostname}}"
         filter: "{{json.event.SourceHostname != null and json.event.SourceHostname != '-'}}"
+
+      - set:
+          source.address: "{{json_event.message.SourceAddr}}"
+        filter: "{{json.event.SourceAddr != null and json.event.SourceAddr != '-'}}"
```

Comment:
```suggestion
```
You can remove this filter as placeholder values will be removed by:
- https://github.com/SEKOIA-IO/intake-formats/pull/815

Reply:
Good to know, I'll remove those